cmd/vulcan-trivy/vulcan-trivy.go

```diff
@@ -118,14 +119,14 @@ func run(ctx context.Context, target string, optJSON string, state state.State)
 		"-f", "json",
 		"-o", reportOutputFile,
 	}
-	// Force vulnerability db cache update.
-	if opt.ForceUpdateDB {
+	// Skip vulnerability db update if not explicitly forced.
+	if !opt.ForceUpdateDB {
```

Comment:
Nitpicking but the db being updated is the default behaviour, and also you are negating this var, so why not use `opt.SkipDBUpdate` instead?

Reply:
As we've discussed, we want to skip vulnerability db as default behaviour.
Default value for a boolean (false) works well and you need to be explicit by setting ForceUpdateDB option if you want to update vulnerability db.
Perhaps we are making a bit confusing the logic but we are making more clear the behaviour and the effect of the options for the check.
Hopefully this has sense for you.